packages/mdc-typography/README.md

```diff
@@ -26,145 +37,69 @@ developers as CSS classes.
-
-#### Style
-
-Simply add the corresponding style class to format your text:
-
-```html
-<body class="mdc-typography">
-  <h1 class="mdc-typography--display4">Big header</h1>
-
-  <p class="mdc-typography--body1">
-    A paragraph with <span class="mdc-typography--body2">emphasis</span>.
-  </p>
-</body>
+npm install --save @material/typography
 ```
 
-The full list of styles:
-
-- `mdc-typography--display4`
-- `mdc-typography--display3`
-- `mdc-typography--display2`
-- `mdc-typography--display1`
-- `mdc-typography--headline`
-- `mdc-typography--title`
-- `mdc-typography--subheading2`
-- `mdc-typography--subheading1`
-- `mdc-typography--body2`
-- `mdc-typography--body1`
-- `mdc-typography--caption`
+## Usage
 
-#### Margins and positioning
+### Load Roboto
 
-In order to minimize unexpected behavior, the style classes only specify font properties, such as size, weight and line
-height.
-
-This means that while text will be correctly styled, it may not be correctly positioned. If you include the
-`mdc-typography--adjust-margin` class, though, positioning will be adjusted according to the style:
+We recommend you load Roboto from Google Fonts
 
 ```html
-<body class="mdc-typography">
-  <h1 class="mdc-typography--display4 mdc-typography--adjust-margin">
-    Big header
-  </h1>
-
-  <p class="mdc-typography--body1 mdc-typography--adjust-margin">
-    A paragraph with
-    <span class="mdc-typography--body2 mdc-typography--adjust-margin">
-      emphasis
-    </span>.
-  </p>
-</body>
+<head>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+</head>
 ```
 
-> Note: as the name implies, `mdc-typography--adjust-margin` will change the margin properties of the element it's
-applied to, in order to align text correctly. Because of this, it should only be used in a text context; using this
-property on UI elements such as buttons may cause them to be positioned incorrectly.
-
+### CSS Classes
 
-## Sass mixin usage
+Some components have a set typographic style. For example, a raised MDC Card uses Body 1, Body 2, and Headline styles.
 
-### mdc-typography-base
+If you want to set the typographic style of an element, which is not a Material Design component, you can apply the following CSS classes.
 
-```scss
-@include mdc-typography-base;
-```
+CSS Class | Description
+--- | ---
+`mdc-typography` | Sets the font to Roboto
+`mdc-typography--display4` | Formats text as Display 4
+`mdc-typography--display3` | Formats text as Display 3
+`mdc-typography--display2` | Formats text as Display 2
+`mdc-typography--display1` | Formats text as Display 1
+`mdc-typography--headline` | Formats text as Headline
+`mdc-typography--title` | Formats text as Title
+`mdc-typography--subheading` | Formats text as Subheading
+`mdc-typography--body2` | Formats text as Body 2
+`mdc-typography--body1` | Formats text as Body 1
+`mdc-typography--caption` | Formats text as Caption
+`mdc-typography--adjust-margin` | Positions text
```

Comment:
Should we re-include something like the original bit that explains "the style mixins only specify font properties" etc though (maybe even just a short parenthetical)?  Otherwise looking at this it's only vaguely evident why I would want to use it.